### DISCUSSION TITLE
Mirth Connect Administrator takes 7 minutes to launch

### DISCUSSION THREAD

**Discussion by JoeFox82** [nextgenhealthcare/connect#5285] [Q&A] [answered]
Hi together,

on some systems launching the Mirth Connect Administrator takes about 7 or more minutes till the login dialog pops up. I'm using Mirth Connect 3.12 with the Mirth Connect Administrator Launcher 1.2.0 and the default Mirth Connect Administrator Launcher settings.
One thing I noticed is that there is a "Internet Connection" popup that I cancel because the server has no internet connection.

Any idea why the start takes this long?

Kind regards,
Jochen

**Comment by pacmano1** [comment]
It is validating jars.   Search both discussions and issues about it.

**Comment by jonbartels** [comment, marked as answer]
https://github.com/nextgenhealthcare/connect/discussions/4944 covers this in detail.

**Comment by JoeFox82** [comment]
Thank you very much, I didn't find something like this with the keywords I entered.
